DSharpPlus.UnifiedCommands/UnifiedCommandsBuilder.cs

```diff
@@ -0,0 +1,110 @@
+using System.Reflection;
+using DSharpPlus.UnifiedCommands.Message;
+using DSharpPlus.UnifiedCommands.Message.Internals;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace DSharpPlus.UnifiedCommands;
+
+public class UnifiedCommandsBuilder
+{
+    private Assembly? _assembly = null;
+    private bool _isAlreadyBuilt = false;
+    private List<string> _prefixes = new();
+    private ulong[]? _guildIds = null;
```

Comment:
it depends on whether they're allowed to stay null. if it can be null then it should be null; there is a semantic difference between "not present" and "empty"